contracts/FlowIDTableStaking_new.cdc

```diff
@@ -1633,6 +1733,14 @@ pub contract FlowIDTableStaking {
         // Save the candidate nodes limit
         FlowIDTableStaking.account.save<Int>(candidateNodeLimit, to: /storage/idTableCandidateNodeLimit)
 
+        let slotCounts: {UInt8: UInt16} = {1: 0, 2: 0, 3: 0, 4: 0, 5: 0}
+
+        // Save slot limits
+        FlowIDTableStaking.account.save(slotCounts, to: /storage/flowStakingSlotLimits)
```

Comment:
I think defaulting to 0 for all node roles here would be a breaking change. For example, integration tests and transient networks like Localnet, Benchnet, and Canary would deploy this as soon as they pin this version of the contract. Then, all subsequent staking operations on those networks would fail.

Because of this, I'm inclined to default to unlimited instead of 0. Then staking will be functional with the default configuration.

Reply:
oh yeah, that is a good point. I was just setting it manually after deploying. I'll change that to be unlimited by default